utils/zetachain.command.helpers.ts

```diff
@@ -66,9 +66,12 @@ export const setupZetachainTransaction = (options: BaseZetachainOptions) => {
 
   let signer: ethers.Wallet;
 
-  const rpc = getRpcUrl(options.network === "mainnet" ? 7000 : 7001);
-
-  if (!rpc) {
+  let rpc;
+  if (options.rpc) {
+    rpc = options.rpc;
+  } else if (options.chainId) {
+    rpc = getRpcUrl(parseInt(options.chainId));
+  } else {
     handleError({
       context: "Failed to retrieve RPC URL",
       error: new Error("RPC URL not found"),
```

Comment:
Since we know the absence of both `rpc` and `chainId` will throw an error, we should add a refine rule to ensure at least one of them is present when running the command, there's no need to even call the `setupZetachainTransaction` function on its callers if this requirement is not met.